mantidproject/mantid
# remove fit benchmarking files

See #22433 

**To test:**

* Check that all tests are passing
* check that all that needs moving has been moved to https://github.com/mantidproject/fitbenchmarking

No release note required
